testsuite/features/core/srv_create_repository.feature

```diff
@@ -50,40 +50,61 @@ Feature: Add a repository to a channel
     And I wait until the channel "test-channel-x86_64" has been synced
     And I disable source package syncing
 
+  Scenario: Add the repository to the x86_64 child channel
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Test-Channel-x86_64 Child Channel"
+    And I enter "file:///etc/pki/rpm-gpg/uyuni-tools-gpg-pubkey-0d20833e.key" as "GPG key URL"
+    And I click on "Update Channel"
+    Then I should see a "Channel Test-Channel-x86_64 Child Channel updated" text
+    When I follow "Repositories" in the content area
+    And I select the "fake-rpm-repo" repo
+    And I click on "Save Repositories"
+    Then I should see a "Test-Channel-x86_64 Child Channel repository information was successfully updated" text
+
+  Scenario: Synchronize the repository in the x86_64 child channel
+    When I enable source package syncing
```

Comment:
Here you enable source package syncing, but you do not disable it again.
From here on, all repos try to mirror source packages